.github/workflows/build.yml

```diff
@@ -21,5 +21,25 @@ jobs:
           java-version: 11
           distribution: 'zulu'
           cache: 'gradle'
+      - name: Extract repository name
+        id: extract_repo_name
+        run: echo ::set-output name=REPO::${GITHUB_REPOSITORY##*/}
       - name: Build
-        run: ./gradlew build --stacktrace
+        run: ./gradlew build --stacktrace ${{ steps.extract_repo_name.outputs.REPO }}-web:installDist"
+      - name: Build slim container image
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: Dockerfile.slim
+          push: true
+          tags: |
+            "${{ env.CONTAINER_REGISTRY }}/${{ steps.extract_repo_name.outputs.REPO }}:spinnaker-${{ env.GITHUB_REF_NAME }}-latest-unvalidated"
```

Comment:
at the top of this file should we add `release-*` to branches to match on?
```
on:
  push:
    branches:
    - master
    - release-*  # build and push "release-x-y-z-latest" on commit to release branch?
    - version-*
```

Reply:
Depending on our naming convention, I suspect so, yes.  I figure we want every push to a release branch to get its own docker image.

Reply:
Thanks, `version-*` is unused so have switched `version-*` to `release-*` in https://github.com/spinnaker/rosco/pull/841/commits/d121964c001902f6680aa38d7a4b0bb9488f6be9

This will build and push a tag akin to a common `:latest` tag except here the tag is `spinnaker-MAJOR.MINOR.x-latest-unvalidated`. 
For example, a git push to `release-1.27.x` branch results in a new `spinnaker-1.27.x-latest-unvalidated` image being built and pushed to GAR - overwriting any existing one.

Will revisit this when understand the patch process and all of the image consumers.
If we want to preserve intermediate container images in-between official "cut" PATCH versions we could add date or commit sha to the tag.

Reply:
I've added both git commit and date time to tag variations in https://github.com/spinnaker/rosco/commit/e47dd63fb60df27701ac0b3d993a5b71afe27280